src/app/components/Registration/assets/facebook-icon.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="34" height="34" viewBox="0 0 34 34" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
delete unnecessary imgs from this folder, pls

Reply:
fixed